frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -601,13 +601,6 @@ export default ({
           sinceHeight = 0
         } else if (this.ephemeral.currentHighestHeight == null || !hasPreviousEvents) {
           const { height: latestHeight } = await sbp('chelonia/out/latestHEADInfo', chatRoomID)
-          /* .finally(() => {
-              if (this.ephemeral.loadingDown === instance) {
-                this.ephemeral.loadingDown = false
-              } else if (this.ephemeral.loadingUp === instance) {
-                this.ephemeral.loadingUp = false
-              }
-            }) */
```

Comment:
it seems this finally block has been moved down L622 below. So removed this comment.